lib/supa/command.rb

```diff
@@ -16,8 +16,12 @@ def represent
     private
 
     def value
-      return @representer.send(@options[:modifier], apply_render_flags(raw_value)) if @options[:modifier]
-      apply_render_flags(raw_value)
+      return @value if defined?(@value)
+      @value = modifier ? @representer.send(modifier, apply_render_flags(raw_value)) : apply_render_flags(raw_value)
```

Comment:
@Strech @dasnotme Do you think it makes sense to add code handle such case? When we pass `''` as `modifier` than it will try to call `@representer.send('', ...)` which will result in `#<NoMethodError: undefined method `' for #<Supa::ArticleRepresenter:0x007fa0d1957918>>`. Another example where `modifier: true` -> `#<TypeError: true is not a symbol nor a string>`.